src/k8s/pkg/utils/cidr.go

```diff
@@ -125,3 +125,51 @@ func ParseCIDRs(CIDRstring string) (string, string, error) {
 	}
 	return ipv4CIDR, ipv6CIDR, nil
 }
+
+// GetLocalhostAddress returns the localhost address based on the given pod and service CIDRs.
+// In IPv6-only mode, the IPv6 localhost address is returned. Otherwise the IPv4 localhost address is returned.
+func GetLocalhostAddress(podCIDR string, serviceCIDR string) (string, error) {
```

Comment:
This feels a bit hacky/magic, what do you think about having a config to explicitly set the preference?

Reply:
discussed offline: let's remove this hacky approach and use the `--address` flag instead